pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
you can't if you use the the git tree hash. As suggested, best may be to move the README to the docs directory and not have it refer to specific hashes. They will be outdated very quick anyway